app/models/school_submission.rb

```diff
@@ -60,26 +60,36 @@
 
 class SchoolSubmission < ActiveRecord::Base
 
-  enum alternate_student: [:yes, :no]
-  enum gender: [:female, :male, :other]
-  enum dietary_restriction: [:dairy_free, :gluten_free, :None, :nut_allergy, :vegan, :vegetarian]
-  enum guardian_phone_type: [:cell, :home, :work]
-  enum guardian_relationship: [
-    :mother,
-    :father,
-    :aunt,
-    :uncle,
-    :grandmother,
-    :grandfather,
-    :stepmother,
-    :stepfather,
-    :guardian,
-  ]
-  enum registration_status: [:registered, :selected, :dropped]
-  enum shirt_size: [:S, :M, :L, :XL, :XXL]
+  enum student_gender: [:female, :male, :other]
```

Comment:
There was an issue where the `student_gender` enum was `gender` instead of `student_gender` - beware of copy pasta errors!